Hymns/Display/VerseView.swift

```diff
@@ -8,15 +8,15 @@ struct VerseLineView: View {
 
     var body: some View {
         HStack(alignment: .top) {
-            Text(viewModel.verseNumber ?? "").font(viewModel.fontSize.font).frame(minWidth: viewModel.fontSize.minWidth)
+            Text(viewModel.verseNumber ?? "").font(viewModel.fontSize.font).frame(minWidth: 12)
```

Comment:
We kind of need a definite size here. This causes the wierd issue especially when you go from big to small text of extra space.

Reply:
I still get the extra space: ![Simulator Screen Shot - iPhone 8 - 2020-06-14 at 17 02 33](https://user-images.githubusercontent.com/1427524/84607211-e167ef80-ae60-11ea-94e4-b565d3e312cd.png)

Reply:
I think we can revert this right?